itzhak/io/schema.py

```diff
@@ -11,9 +11,13 @@ class Meta:
 
 class Query(graphene.ObjectType):
     users = graphene.List(UserType)
+    user = graphene.Field(UserType, id=graphene.ID())
```

Comment:
```suggestion
    user = graphene.Field(UserType, name=graphene.String())
```
@salitosmbogz ♻️ About the `name` vs. `id` comment.

Reply:
♻️